host/tests/L3_TestCases/dsFPD/dsFPD_test01_EnableDisableAndVerifyLEDIndicators.py

```diff
@@ -91,5 +91,7 @@ def testFunction(self):
         return result
 
 if __name__ == '__main__':
+    summerLogName = os.path.splitext(os.path.basename(__file__))[0] + "_summery"
+    summeryLog = logModule(summerLogName, level=logModule.INFO)
     test = dsFPD_test01_EnableDisableAndVerifyLEDIndicators()
```

Comment:
summeryLog variable is not used anywhere.. it should be passed in line 96

Reply:
SummaryLog object is passed to test object.